roosevelt.js

```diff
@@ -44,6 +47,19 @@ module.exports = function(params) {
 
   console.log(('💭  ' + 'Starting ' + app.get('appName') + ' in ' + app.get('env') + ' mode...').bold);
 
+  // check if env comes from cmd line then leave as is, else use params
```

Comment:
as i understand it the hierarchy of expected behavior is, in order of priority:

1. command line flag for dev/prod (currently available, triggered by `npm run dev` and `run prod` commands)
2. constructor param (currently unavailable, this PR implements this, intended to supersede `NODE_ENV`, but not command line flags)
3. package.json param  (currently unavailable, this PR implements this, intended to supersede `NODE_ENV`, but not command line flags)
4. `NODE_ENV` environment variable (currently available, intended to be what decides this if no command line flags or params are supplied)